Core.ApplicationServices/IEventBody.cs

```diff
@@ -1,8 +1,6 @@
-﻿using System.Collections.Generic;
-
-namespace Core.ApplicationServices;
+﻿namespace Core.ApplicationServices;
 
 public interface IEventBody
 {
-    Dictionary<string, object> ToKeyValuePairs();
+    string Type();
```

Comment:
I think GetType would be a better name

Reply:
Or even better GetTypeName - GetType sounds more like it's going to return e.g. "SystemNameChangeEventBodyModel"

Reply:
Also, consider moving the topic creation into the Model.
It can handle it in a GetTopic(string topic), and call on the private GetTypeName method